IPython/core/magics/code.py

```diff
@@ -276,7 +276,7 @@ def pastebin(self, parameter_s=''):
         ).encode("utf-8")
 
         request = Request(
-            "http://dpaste.com/api/v2/",
+            "https://dpaste.com/api/v2/",
```

Comment:
That should be a separate PR :). Please don't mix these two things.

Reply:
Yes, very sorry. I thought that i cancelled the second PR. I will do so right away!